frontend/templates/views/2021/partials/content/link.j2

```diff
@@ -0,0 +1,10 @@
+<a class="ap-link{% if external %} --external{% else %} --internal{% endif %}{% if square %} --square{% endif %}{% if icon %} --icon{% endif %}"
+  href="{{ href }}">
+  {% if icon %}
+    <div class="ap-icon ap-link-icon">
```

Comment:
Do we ever use links in this form with an icon other than the specified errors?

Reply:
I think you mean arrows, not errors? :) 
I guess I was mislead, since the class ```ap-m-lnk-icon``` is used in other parts of the page (e.g. https://amp.dev/community/working-groups/infra/ -> channel links in the communication channels section).

I will remove it.

Reply:
Whoops, yes 😁 thanks!